src/Client/Web/Bit.Client.Web.BlazorUI/Components/Buttons/BitButton.razor.cs

```diff
@@ -23,13 +23,13 @@ public ButtonStyle ButtonStyle
             }
         }
 
-        protected override string RootElementClass => "bit-button";
+        protected override string RootElementClass => "bit-btn";
 
         protected override void RegisterComponentClasses()
         {
             ClassBuilder.Register(() => IsEnabled is false ? string.Empty :
-                                        ButtonStyle == ButtonStyle.Primary ? "primary" :
-                                        "standard");
+                ButtonStyle == ButtonStyle.Primary ? $"{RootElementClass}-primary-{VisualClassRegistrar()}"
+                : $"{RootElementClass}-standard-{VisualClassRegistrar()}");
```

Comment:
improve this ternary operator code formatting

Reply:
Fixed all ternary  operators formatting